### ISSUE TITLE
the way to get FID, PSNR and SSIM on  semantic image synthesis???

### ISSUE THREAD

**Issue by loseway12138** [Xiaoming-Yu/DMIT#9]
In the semantic image synthesis task, how to select datas to calculate FID, PSNR and SSIM indicators

**Comment by Xiaoming-Yu** [comment]
Hi,
the semantic image synthesis task is evaluated on CUB dataset.
We divide the CUB dataset into training (8,855) and test sets(2,933) according to the method in AttnGAN.
As for the the FID, we first randomly sample 2,500 images and mismatched texts from test set to generate new syntheses. Then we compute the FID score between the distribution of generated images and training set.
As for the PSNR and SSIM, we translate each test image by its corresponding text, and calculate the psnr and ssim between the input and generated images.

**Comment by loseway12138** [comment]
Thanks for your reply，can you share the relevant code？

**Comment by loseway12138** [comment]
Hi, I still have some unclear points. Do you mean to calculate the PSNR and SSIM of the test images and their reconstructed images?

**Comment by Xiaoming-Yu** [comment]
Yes,  PSNR and SSIM are used to measure the difference between the image and its reconstruction.
( I forgot to backup related test codes when I graduated ==

**Comment by loseway12138** [comment]
It's okay, thank you for your reply.
